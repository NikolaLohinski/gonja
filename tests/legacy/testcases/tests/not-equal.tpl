{{ 42 is ne 42 }}
{{ 42 is != 42 }}
{{ 42 is ne 3 }}
{{ 42 is != 3 }}
{{ 'gonja' is ne 'jinja' }}
{{ 'gonja' is ne 'gonja' }}
