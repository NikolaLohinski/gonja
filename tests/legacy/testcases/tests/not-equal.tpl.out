False
False
True
True
True
False
